deployment/compose/grafana/grafana.ini

```diff
@@ -656,7 +656,7 @@ org_role = Admin
 # Use space to separate multiple modes, e.g. "console file"
 ;mode = console file
 
-# Either "debug", "info", "warn", "error", "critical", default is "info"
+# Either "debug", utils.InfoStr, "warn", utils.ErrorStr, "critical", default is utils.InfoStr
```

Comment:
I believe we mistakenly refactored everything, including this guy?

Reply:
yep, my bad, my sed was to big, will fix it. thanks